VideoLocker/src/main/java/org/edx/mobile/view/CourseDiscussionPostsActivity.java

```diff
@@ -74,6 +74,11 @@ public Fragment getFirstFragment() {
         } else {
             fragment = courseDiscussionPostsThreadFragment;
         }
+        if (fragment.getArguments() == null) {
+            Bundle args = new Bundle();
+            args.putSerializable(Router.EXTRA_ENROLLMENT_COURSE_DATA, courseData);
+            fragment.setArguments(args);
```

Comment:
This seems to be a bigger change, that'll require more refactoring than we expect. So, I'll let it pass.